lib/src/components/NxToast/NxToast.scss

```
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @use '../../scss-shared/nx-header-variables';
+
+ // 74px refers to the total height of each toast: the content itself, plus the padding
+ // and margin-top applied. 
+ $-nx-toast-height: 74px;
```

Now that the margin is on `.nx-toast` and not `.nx-alert`, this should be reduced to 66px
```suggestion
 $-nx-toast-height: 66px;
```

---

> Now that the margin is on `.nx-toast` and not `.nx-alert`, this should be reduced to 66px

Toast height updated here: [251c518](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/251c518f75cdfe5952c1f4f71346111491d3add8)